generators/gae/templates/app.yaml.ejs

```diff
@@ -28,3 +50,4 @@ automatic_scaling:
   max_instances: <%= gaeMaxInstances %>
   <%_ } _%>
 <%_ } _%>
+entrypoint: java -agentpath:/opt/cdbg/cdbg_java_agent.so=--log_dir=/var/log -noverify -XX:+AlwaysPreTouch -XX:TieredStopAtLevel=1 -Djava.security.egd=file:/dev/./urandom -cp BOOT-INF/resources/:BOOT-INF/classes/:BOOT-INF/lib/* <%= packageName %>.<%= mainClass %>
```

Comment:
quick que - where was mainClass specified? I ran into an issue on this PR branch:

```
Creating Google App Engine deployment files
ERROR! Error: Copying template app.yaml.ejs failed. [ReferenceError: /Users/rayt/Projects/git/generator-jhipster/generators/gae/templates/app.yaml.ejs:53
    51| <%_ } _%>
    52| <%_ } _%>
 >> 53| entrypoint: java -agentpath:/opt/cdbg/cdbg_java_agent.so=--log_dir=/var/log -noverify -XX:+AlwaysPreTouch -XX:TieredStopAtLevel=1 -Djava.security.egd=file:/dev/./urandom -cp BOOT-INF/resources/:BOOT-INF/classes/:BOOT-INF/lib/* <%= packageName %>.<%= mainClass %>
    54| 

mainClass is not defined]
```

Thanks!

Reply:
@saturnism : Ah, my mistake, I didn't check properly. I've just pushed a change that fixes this. :smile: